### PR TITLE
docs: fix manpage for configuration file

### LINKED ISSUE #329: Wrong manpage name for configuration file
Manpage for `storage.conf` is being built as `container-storage.conf.5` (notice the missing **s** after `container`), but it should be `containers-storage.conf.5` for consistency with the whole project naming.

### PR DESCRIPTION
**Pull request by Silvanoc**
Manpage for storage.conf is being built as 'container-storage.conf.5'
(notice the missing 's' after 'container'), but it should be
'containers-storage.conf.5' for consistency with the whole project
naming.

Closes #329

Signed-off-by: Silvano Cirujano Cuesta <silvano.cirujano-cuesta@siemens.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
@nalind @TomSweeneyRedHat @vrothberg PTAL

File: docs/Makefile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 GOMD2MAN = go-md2man
 
-docs: $(patsubst %.md,%.1,$(wildcard *.md)) container-storage.conf.5
+docs: $(patsubst %.md,%.1,$(wildcard *.md)) containers-storage.conf.5
```
Comment (vrothberg) [inline docs/Makefile]: The wildcard still has one issue -> `containers-storage.conf.5.1`. Would you fix that as well?

Comment (Silvanoc) [reply docs/Makefile]: Sure

**Comment by Silvanoc** [comment]
I've added commit https://github.com/containers/storage/pull/330/commits/35217acd13c7013d7a1fa7cbc377e8e376e5a333 to fix the issue mentioned by @vrothberg [in this comment](https://github.com/containers/storage/pull/330#discussion_r278872325) and another issue that I've found in the `clean` target of the global `Makefile`.

Although this commit handles something different from the original reason for the PR, it's related and all together the PR remains very small and easy to review. Therefore putting it in the same PR.

**Comment by vrothberg** [comment]
LGTM, thanks a lot!

> Although this commit handles something different from the original reason for the PR, it's related and all together the PR remains very small and easy to review. Therefore putting it in the same PR.

Thanks. I prefer having one PR than multiple ones as I find it easier to manage. It also safes some energy as less CI is needed.

**Comment by rhatdan** [comment]
Thanks @Silvanoc

**Pull request merged**